### DISCUSSION TITLE
Make paused state observation in NSF optional

### DISCUSSION THREAD

**Discussion by Richard-L** [OpenRCT2/OpenRCT2#17856] [Feature requests]
Would you consider making it optional whether saving to `.park` regards the park's pause state? The old behaviour of a park being always unpaused on load has certain advantages that I miss:
- To time effects like explosions of crashing trains, or duelling coasters performing a specific element right as you load the park.
- To hear the beep and be greeted with a custom message printing (if one has been added).
- Bit of an NE concern: since we remove all unused objects before releasing parks, we sadly need to overwrite the submitted file. To not advance any ticks (so as not to alter the moment when the submitter saved), we need to save whilst the game is paused (after having run `remove_unsued_objects`). With the NSF this now means the park is inevitably paused, even if a submitter didn't intend for it.

Thanks in advance

**Comment by frutiemax** [comment]
I don't think it would hurt to add a single flag to the park save format since many people seem to want to have game paused when loading the park.

**Reply by autosysops** [reply]
> I don't think it would hurt to add a single flag to the park save format since many people seem to want to have game paused when loading the park.

I think it might be even doable if for example a console command was added where you can save the game and set the state to unpaused at the same time. In that case no flags need to be added, it just needs to overwrite the current state with an active state when saving.

**Reply by Richard-L** [reply]
Oh that could be a nice solution

**Reply by thiebert** [reply]
Would appreciate the option to save either way. I often pause before a save to time events upon opening, but don't want to force others to unpause first. Being able to save while paused, but have it open unpaused would be a good option!

Alternatively, while working on hacks and such, keeping the pause state when opening has a lot of value. A console command or alternate menu option would be helpful here!
